### PR TITLE
Add typing to I2C bus

### LINKED ISSUE #11: Missing Type Annotations
There are missing type annotations for some functions in this library.

The `typing` module does not exist on CircuitPython devices so the import needs to be wrapped in try/except to catch the error for missing import. There is an example of how that is done here:

```python
try:
    from typing import List, Tuple
except ImportError:
    pass
```

Once imported the typing annotations for the argument type(s), and return type(s) can be added to the function signature. Here is an example of a function that has had this done already:

```python
def wrap_text_to_pixels(
    string: str, max_width: int, font=None, indent0: str = "", indent1: str = ""
) -> List[str]:
```
If you are new to Git or Github we have a guide about contributing to our projects here: https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github

There is also a guide that covers our CI utilities and how to run them locally to ensure they will pass in Github Actions here: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code In particular the pages: `Sharing docs on ReadTheDocs` and `Check your code with pre-commit` contain the tools to install and commands to run locally to run the checks.

If you are attempting to resolve this issue and need help, you can post a comment on this issue and tag both @foamyguy and @kattni or reach out to us on Discord: https://adafru.it/discord in the `#circuitpython-dev` channel.

The following locations are reported by mypy to be missing type annotations:

- [ ] adafruit_monsterm4sk.py:65

### PR DESCRIPTION
**Pull request by SebastienFauque**
Adds type annotations to MonsterM4sk I2C bus. Solves issue https://github.com/adafruit/Adafruit_CircuitPython_MONSTERM4SK/issues/11

### CODE REVIEW

File: adafruit_monsterm4sk.py [hunk 1]
```diff
@@ -62,7 +68,7 @@ class MonsterM4sk:
     The right screen is the one USB port directly above it.
     """
 
-    def __init__(self, i2c=None):
+    def __init__(self, i2c: Optional[I2CDeviceDriver] = None):
```
Comment (FoamyGuy) [inline adafruit_monsterm4sk.py]: I think `i2c` can be the `I2C` from `busio` like this:

```
from busio import I2C
# ....
def __init__(self, i2c: Optional[I2C] = None):
```

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
Thanks for working on this @SebastienFauque!

One tweak to the type used for I2C.

I've also noticed that the original issue seems to have not listed all of the relavent lines of code with functions that can be typed. 

If you're up to it, it'd be great to also add types for the function arguments and return types for:
- acceleration
- light
- buttons
- boop

If you're up for that but need help figuring out the proper types let me know I can assist with that. 

If you'd rather not do those ones, it's okay as well once this one is good to go we can merge and I'll keep the issue open and update it with the rest of the lines.

File: adafruit_monsterm4sk.py (hunk 1 above)
Comment (SebastienFauque) [reply adafruit_monsterm4sk.py]: @FoamyGuy Thanks for the tip, changes have been made.

**Review by FoamyGuy — APPROVED** [review]
I added a commit with one tweak to swap the order of the imports inside the try block so that the typing one comes first which will raise the exception when it runs on the microcontroller before it's imported anything else.

Latest version looks good to me. @SebastienFauque Thanks again for working on this and taking care of the other functions that weren't listed in the original issue!

**Pull request merged**